lib/api/dsl/storage/index.js

```diff
@@ -431,7 +445,7 @@ function addIndexCollectionToObject(obj, index, collection) {
  * @param {object} filter - filter to be removed
  */
 function removeFromTestTables(filters, subfilters, testTables, index, collection, filter) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 91 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)